source/upgrade.txt

```diff
@@ -76,6 +76,18 @@ Version 6.x Breaking Changes
 - Automatic Encryption methods are now in the {+driver-short+}. You must
   import these methods from the driver instead of from
   ``mongodb-client-encryption``.
+- Removed support for strings with a length of 12 from the ``ObjectId``
+  constructor.
+- The ``abortTransaction`` and ``commitTransaction`` methods return
+  ``null`` instead of the raw command results.
+- Removed connection option helpers that recognized values other than ``true``
+  or  ``false`` as booleans. You must provide either ``true`` or ``false`` values for
+  options that expect boolean values in the connection string or MongoClient
+  constructor.
+- Removed the Binary BSON type constructor that accepted a string.
+- The ``Binary.write`` method no longer accepts a string to write to the binary
+  BSON object.
+- The ``ClientEncryption`` API returns promises instead of callbacks.
```

Comment:
Suggestion:

I think we use monospace formatting for [a number of types of items according to the style guide](https://www.mongodb.com/docs/meta/style-guide/style/text-formatting/), but have not seen it used for names of APIs. One example of an API we refer to often is "Stable API" which does not use monospace.

Reply:
Changed it to be non-monospaced.